### PR TITLE
Adding Sam and Becky, removing Jordan and Michael Parrish

### LINKED ISSUE #2578: Update team page
Related to #2575, we have lots of new people! Let's get the team page updated. @mcbouslog, @amyrebecca, @zwolf, @wgranger could you write up a short 2-3 sentence bio and pick out a nice photo for a mugshot? @mrniaboc do you know who else may be missing?

### PR DESCRIPTION
**Pull request by beckyrother**
Fixes #2578.

Added Sam and Becky, removed Jordan and Michael Parrish from team list

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

File: app/pages/about/team-page.cjsx [hunk 1]
```diff
@@ -211,6 +214,9 @@ counterpart.registerTranslations 'en',
         title: 'Developer'
         bio: '''Roger builds front end interfaces for the Zooniverse, and enjoys coding, music and rebuilding
         motorbikes in his spare time.'''
+      samanthaBlickham:
+        title: 'IMLS Postdoctoral Fellow'
+        bio: '''Samantha Blickhan is the IMLS Postdoctoral Fellow in the Department of Citizen Science at the Adler Planetarium, working on transcription projects for the Zooniverse. She received her Ph.D. in Musicology from Royal Holloway, University of London, with a thesis on the palaeography of British song notation in the 12th and 13th centuries. Her research interests include music and perception, and their relationships with writing systems, technology and pedagogy..'''
```
Comment (eatyourgreens) [inline app/pages/about/team-page.cjsx]: Typo at the end here: extra full stop character.

File: app/pages/about/team-page.cjsx [hunk 2]
```diff
@@ -128,12 +131,12 @@ counterpart.registerTranslations 'en',
         bio: '''Jen works on Galaxy Zoo education in her role as the SEPnet/Ogden Outreach Officer
         for the Institute of Cosmology and Gravitation, and has a PhD in astrophysics. Active
         galactic nuclei were her first love.'''
-      jordanMartin:
-        title: 'Visual Designer'
-        bio: '''Jordan joined at the Adler Planetarium as Zooniverse's Visual Designer
-        starting in 2015. She spends her time making our user interfaces beautiful,
-        engaging, and a rewarding experience. At home she spends time with her
-        two cats and many houseplants.'''
+      # jordanMartin:
```
Comment (eatyourgreens) [inline app/pages/about/team-page.cjsx]: I think you can delete these sections, rather than commenting them out.

**Comment by eatyourgreens** [comment]
This adds the English translations for the new people, but they also need to be added to the list of team members. That's further down the file, and the entries look something like this:
```
jimODonnell:
    name: "Jim O'Donnell"
    twitter: "pekingspring"
    title: counterpart "team.content.jimODonnell.title"
    bio: counterpart "team.content.jimODonnell.bio"
    image: "/assets/team/jim.jpg"
    location: "oxford"
```
The jpeg files for the images should be added to the public folder, in `/public/assets/team/`.

File: app/pages/about/team-page.cjsx (hunk 2 above)
Comment (marten) [inline app/pages/about/team-page.cjsx]: We're moving them to the alumni section and keeping them I think.

**Comment by beckyrother** [comment]
Fixed typo, added images, moved Beverly to alumni. @trouille, should Michael and Jordan be moved to alumi as well? Right now we just removed them.

**Comment by trouille** [comment]
Yes, please do move Michael and Jordan to alum rather than remove. Thanks!

**Comment by beckyrother** [comment]
Updated to move Jordan and Michael to alumni.

File: app/pages/about/team-page.cjsx [hunk 3]
```diff
@@ -211,6 +205,9 @@ counterpart.registerTranslations 'en',
         title: 'Developer'
         bio: '''Roger builds front end interfaces for the Zooniverse, and enjoys coding, music and rebuilding
         motorbikes in his spare time.'''
+      samanthaBlickham:
```
Comment (srallen) [inline app/pages/about/team-page.cjsx]: This should be `samanthaBlickhan`

File: app/pages/about/team-page.cjsx [hunk 4]
```diff
@@ -320,7 +317,14 @@ teamMembers =
     title: counterpart "team.content.beverleyNewing.title"
     bio: counterpart "team.content.beverleyNewing.bio"
     image: "/assets/team/beverley.jpg"
-    location: "oxford"
+    location: "alumni"
+beckyRother:
```
Comment (srallen) [inline app/pages/about/team-page.cjsx]: The indent on this one is off. Like I mentioned, coffeescript doesn't like misaligned indents. This needs to be one indents (2 spaces).

**Comment by srallen** [comment]
@beckyrother oh! and a couple more things is that Jordan and Michael's bios and title didn't get added back in. Let me know if you want any more help with this. Thanks!

**Pull request merged**